### PR TITLE
TOAST MESSAGES

### PR DESCRIPTION
**Pull request by leonel-acosta**

### CODE REVIEW

File: js-create-element/js-createelement_toast-messages/js/index.js [hunk 1]
```diff
@@ -4,10 +4,22 @@ const toastContainer = document.querySelector('[data-js="toast-container"]');
 const addButton = document.querySelector('[data-js="add-button"]');
 const clearButton = document.querySelector('[data-js="clear-button"]');
 
-addButton.addEventListener("click", () => {
+addButton.addEventListener("click", (event) => {
   // Exercise: Append a new entry to the toast messages container
+  event.preventDefault();
```
Comment (fadiktekin) [inline js-create-element/js-createelement_toast-messages/js/index.js:9]: `event.preventDefault()` is not needed here I think. If this was a `form` and we used `submit` event, we would add `preventDefault` for the form not to do GET request and stop the reload

File: js-create-element/js-createelement_toast-messages/js/index.js [hunk 2]
```diff
@@ -4,10 +4,22 @@ const toastContainer = document.querySelector('[data-js="toast-container"]');
...
+    <p>New Message</p>`;
+  toastContainer.append(newMessage);
+  console.log("New message added")
 });
 
-clearButton.addEventListener("click", () => {
+clearButton.addEventListener("click", (event) => {
   // Exercise: Clear the stack of toast messages
+  event.preventDefault();
+  const newMessage = document.createElement("li");
```
Comment (fadiktekin) [inline js-create-element/js-createelement_toast-messages/js/index.js:21]: This 2 lines are not needed

File: js-create-element/js-createelement_toast-messages/js/index.js (hunk 1 above)
Comment (leonel-acosta) [reply js-create-element/js-createelement_toast-messages/js/index.js]: fixed

File: js-create-element/js-createelement_toast-messages/js/index.js (hunk 2 above)
Comment (leonel-acosta) [reply js-create-element/js-createelement_toast-messages/js/index.js]: fixed

**Pull request merged**